test/integration/test-plugin/src/index.ts

```diff
@@ -1,22 +1,26 @@
+// TODO@jsjoeio - how do I fix this?
+// @ts-ignore - we know code-server exists
```

Comment:
🤔 not sure why this wasn't happening before I move it. Maybe I need to `tsconfig.json` ignore the test plugin? Any ideas?

Reply:
Might need to update the path here: https://github.com/coder/code-server/blob/f9b3bda9d5ce0e3185b7dfd9205615871302e1d5/test/integration/test-plugin/tsconfig.json#L47